test_nettoyage.py

```diff
@@ -0,0 +1,296 @@
+    """Classe permettant de tester le module nettoyage"""
+
+    def test_check_reference_files(self):
+        json_test = {"key1": "departement2020.csv",
+                     "key2": "decp.json",
+                     "path_to_data": "data"
+                     }
+        value = nettoyage.check_reference_files(json_test)
+        assert(not value)  # value doit etre egal à None
+        json_test2 = {"path_to_data": "data",
+                      "key1": "NoFile"
+                      }
+        with pytest.raises(ValueError):
+            nettoyage.check_reference_files(json_test2)
+
+    def test_prise_en_compte_modifications(self):
+        test_no_colonne_modification = pd.DataFrame([1], columns=["test"])
+        test_with_colonne_modification_input = pd.DataFrame([[1, [{"col1Modification": "2",
+                                                                  "col2": "3"
+                                                                   }]
+                                                              ],
+                                                             [2, []]
+                                                             ], columns=["id", "modifications"])
+        test_with_colonne_modification_output = pd.DataFrame([[1, [{"col1Modification": "2",
+                                                                    "col2": "3"
+                                                                    }
+                                                                   ], 1, "2", "3"
+                                                               ],
+                                                              [2, [], 0, "", ""]
+                                                              ], columns=["id", "modifications", "booleanModification", "col1Modification", "col2Modification"])
+        # Test du message d'erreur
+        with pytest.raises(ValueError):
+            nettoyage.prise_en_compte_modifications(test_no_colonne_modification)
+        nettoyage.prise_en_compte_modifications(test_with_colonne_modification_input)
+        assert_frame_equal(test_with_colonne_modification_input, test_with_colonne_modification_output)
+
+    def test_fusion_source_modification(self):
+        df_input = pd.DataFrame([["A", "B", "C", "D", "E"],
+                                 ["F", "G", "H", "I", "J"],
+                                 ["K", "L", "M", "N", ""]
+                                 ], columns=["col1", "col2", "col3", "col1Modification", "col3Modification"])
+        raw = df_input.iloc[0]
+        raw2 = df_input.iloc[2]
+        col_modification = ["col1Modification", "col3Modification"]
+        dic_modif = {"col1Modification": "col1",
+                     "col3Modification": "col3"
+                     }
+        _ = nettoyage.fusion_source_modification(raw, df_input, col_modification, dic_modif)
+        _ = nettoyage.fusion_source_modification(raw2, df_input, col_modification, dic_modif)
```

Comment:
Pourquoi l'assigner si on ne l'utilise pas ?
```suggestion
        nettoyage.fusion_source_modification(raw, df_input, col_modification, dic_modif)
        nettoyage.fusion_source_modification(raw2, df_input, col_modification, dic_modif)
```